src/server/test/web/readingsData/expected_bar_ri_15_mu_kWh_gu_kWh_st_-inf_et_inf_bd_75.csv

```diff
@@ -0,0 +1,2 @@
+reading,start time,end time
```

Comment:
Not sure why this is listed as a change but same as current development so should be fine. Just noting.